### ISSUE TITLE
accidental bsv to bitcoin.com bch wallet

### ISSUE THREAD

**Issue by tveraemt061** [Bitcoin-com/Wallet#570]
I recently have been back and forth via email with customer support at bitcoin.com, the customer service stated they can not help me in this matter any further.
In January I sent BSV to bitcoin.com BCH wallet.  The transaction shows on blockchair its confirmed address received funds and are unspent.
My android bch wallet has a private key which is in text/file and or the 12 word recovery phrase
I used the master seed/ recovery to back up my wallet in Exodus which supports BSV wallets and BCH wallets, also I recovered in electrumsv wallet. I tried to sweep, and recover/search for funds.
The request on electrum is the master private key for the transaction linked to the bch address.
My concern is I have in my wallet address, view all area MY BCH Address I used with BSV monies sitting, non touched. I really need to gain control over this transaction. Is there a way that bitcoin.com will soon support BSV where I can reload/refresh/or Re-broadcast to receive lost funds. or Implement now a re broadcast feature for unspent & spent transactions when restored on a new device.

**Comment by Bitkilo** [comment]
Hello, 
The Bitcoin.com wallet has no plans to support BSV anytime soon.
As for the tx re-broadcasting, this is not something i think is needed in our wallet. 

Have you tried asking this recovery question on the BSV sub-reddit yet, someone there has probably made this same mistake and should be able to help you.

Best of luck!

**Comment by tveraemt061** [comment]
Yes been asking around. I downloaded different bch and bsv wallets in order
to restore with private seed bsv. And electrumsv. Electroncash. Hodler,
exodus, btc all sorts of extraction tools and nothing is helping re
broadcast my used address used to accidentally collect bsv from my
bitcoin.com bch wallet... so frustrating that bitcoin dont give wif address
for recovering its wallets yet for paper wallet it does give wif.. the seed
I have that is my backup and restore for wallet is not working.
